zt_frontend/src/components/CodeComponent.vue

```diff
@@ -513,6 +518,20 @@ export default {
     renameCell(e: String) {
       this.cellData.cellName = e as string;
     },
+    handleFocus() {
+    this.isFocused = true;
+    this.runLint = true;
+    if (this.view) {
+      this.view.dispatch({}); // Force the linter to re-run
```

Comment:
This shouldn't be necessary I think. Simply setting this.runLint to true should force the linter to rerun